make buildserverall errors in conformance

We are trying to implement a plugin for gogo/protobuf. A `make buildserverall` is failing for us in current master branch 06ec6c31ff1bac6ed4e205a547a3d72934813ef3.

<details>
  <summary>Make output</summary>
<pre>
<code>
> make buildserverall                                                                                                   feature/enum_customtype [06ec6c31]
go clean ./...
go install ./proto
go install ./gogoproto
go install ./jsonpb
go install ./protoc-gen-gogo
go install ./protoc-gen-gofast
go install ./protoc-gen-gogofast
go install ./protoc-gen-gogofaster
go install ./protoc-gen-gogoslick
go install ./protoc-gen-gostring
go install ./protoc-min-version
go install ./protoc-gen-combo
go install ./gogoreplace
make -C protoc-gen-gogo/descriptor regenerate
go install github.com/gogo/protobuf/protoc-gen-gogo
go install github.com/gogo/protobuf/protoc-gen-gostring
protoc --gogo_out=. -I=../../protobuf/google/protobuf ../../protobuf/google/protobuf/descriptor.proto
protoc --gostring_out=. -I=../../protobuf/google/protobuf ../../protobuf/google/protobuf/descriptor.proto
make -C protoc-gen-gogo/plugin regenerate
go install github.com/gogo/protobuf/protoc-gen-gogo
protoc --gogo_out=Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor:. -I=../../protobuf/google/protobuf/compiler/:../../protobuf/ ../../protobuf/google/protobuf/compiler/plugin.proto
make -C protoc-gen-gogo/testdata regenerate
go install github.com/gogo/protobuf/protoc-min-version
protoc-min-version --version="3.0.0" --gogo_out=. ./my_test/test.proto
make -C gogoproto regenerate
go install github.com/gogo/protobuf/protoc-gen-gogo
protoc --gogo_out=Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor:. --proto_path=../../../../:../protobuf/:. *.proto
make -C proto/testdata regenerate
go install github.com/gogo/protobuf/protoc-min-version
protoc-min-version --version="3.0.0" --gogo_out=. test.proto
make -C jsonpb/jsonpb_test_proto regenerate
protoc-min-version --version="3.0.0" --gogo_out=Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types:. *.proto -I . -I ../../ -I ../../protobuf/
make -C _conformance regenerate
protoc-min-version --version="3.0.0" --gogo_out=\
	Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,\
	Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,\
	Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types,\
	Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,\
	Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,\
	Mgoogle/protobuf/field_mask.proto=github.com/gogo/protobuf/types\
	:. conformance_proto/conformance.proto
google/protobuf/any.proto: File not found.
google/protobuf/duration.proto: File not found.
google/protobuf/field_mask.proto: File not found.
google/protobuf/struct.proto: File not found.
google/protobuf/timestamp.proto: File not found.
google/protobuf/wrappers.proto: File not found.
conformance_proto/conformance.proto: Import "google/protobuf/any.proto" was not found or had errors.
conformance_proto/conformance.proto: Import "google/protobuf/duration.proto" was not found or had errors.
conformance_proto/conformance.proto: Import "google/protobuf/field_mask.proto" was not found or had errors.
conformance_proto/conformance.proto: Import "google/protobuf/struct.proto" was not found or had errors.
conformance_proto/conformance.proto: Import "google/protobuf/timestamp.proto" was not found or had errors.
conformance_proto/conformance.proto: Import "google/protobuf/wrappers.proto" was not found or had errors.
conformance_proto/conformance.proto:221:3: "google.protobuf.BoolValue" is not defined.
conformance_proto/conformance.proto:222:3: "google.protobuf.Int32Value" is not defined.
conformance_proto/conformance.proto:223:3: "google.protobuf.Int64Value" is not defined.
conformance_proto/conformance.proto:224:3: "google.protobuf.UInt32Value" is not defined.
conformance_proto/conformance.proto:225:3: "google.protobuf.UInt64Value" is not defined.
conformance_proto/conformance.proto:226:3: "google.protobuf.FloatValue" is not defined.
conformance_proto/conformance.proto:227:3: "google.protobuf.DoubleValue" is not defined.
conformance_proto/conformance.proto:228:3: "google.protobuf.StringValue" is not defined.
conformance_proto/conformance.proto:229:3: "google.protobuf.BytesValue" is not defined.
conformance_proto/conformance.proto:231:12: "google.protobuf.BoolValue" is not defined.
conformance_proto/conformance.proto:232:12: "google.protobuf.Int32Value" is not defined.
conformance_proto/conformance.proto:233:12: "google.protobuf.Int64Value" is not defined.
conformance_proto/conformance.proto:234:12: "google.protobuf.UInt32Value" is not defined.
conformance_proto/conformance.proto:235:12: "google.protobuf.UInt64Value" is not defined.
conformance_proto/conformance.proto:236:12: "google.protobuf.FloatValue" is not defined.
conformance_proto/conformance.proto:237:12: "google.protobuf.DoubleValue" is not defined.
conformance_proto/conformance.proto:238:12: "google.protobuf.StringValue" is not defined.
conformance_proto/conformance.proto:239:12: "google.protobuf.BytesValue" is not defined.
conformance_proto/conformance.proto:241:3: "google.protobuf.Duration" is not defined.
conformance_proto/conformance.proto:242:3: "google.protobuf.Timestamp" is not defined.
conformance_proto/conformance.proto:243:3: "google.protobuf.FieldMask" is not defined.
conformance_proto/conformance.proto:244:3: "google.protobuf.Struct" is not defined.
conformance_proto/conformance.proto:245:3: "google.protobuf.Any" is not defined.
conformance_proto/conformance.proto:246:3: "google.protobuf.Value" is not defined.
conformance_proto/conformance.proto:248:12: "google.protobuf.Duration" is not defined.
conformance_proto/conformance.proto:249:12: "google.protobuf.Timestamp" is not defined.
conformance_proto/conformance.proto:250:12: "google.protobuf.FieldMask" is not defined.
conformance_proto/conformance.proto:251:12: "google.protobuf.Struct" is not defined.
conformance_proto/conformance.proto:252:12: "google.protobuf.Any" is not defined.
conformance_proto/conformance.proto:253:12: "google.protobuf.Value" is not defined.

panic: exit status 1

goroutine 1 [running]:
panic(0xc4e20, 0xc4200109c0)
	/usr/local/opt/go/libexec/src/runtime/panic.go:500 +0x1a1
main.main()
	/Users/serabe/golang/src/github.com/gogo/protobuf/protoc-min-version/minversion.go:63 +0x242
make[1]: *** [regenerate] Error 2
make: *** [regenerate] Error 2
</code>
</pre>
</details>

Let us remark that this is without any modification on our side. What are we doing wrong?

Thank you!

--- a/_conformance/Makefile
+++ b/_conformance/Makefile
@@ -30,7 +30,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 regenerate:
-	protoc-min-version --version="3.0.0" --gogo_out=\
+	protoc-min-version --version="3.0.0" --proto_path=$(GOPATH)/src:$(GOPATH)/src/github.com/gogo/protobuf/protobuf:. --gogo_out=\
 	Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,\
 	Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,\
 	Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types,\

